ai_bridge/models/ai_bridge_execution.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo import _, fields, models
+
+
+class AiBridgeExecution(models.Model):
+
+    _name = "ai.bridge.execution"
+    _description = "Ai Execution"  # TODO
+
+    ai_bridge_id = fields.Many2one(
+        "ai.bridge",
+        required=True,
+        ondelete="cascade",
+    )
+    res_id = fields.Integer(required=True)
+    model_id = fields.Many2one(
+        "ir.model",
+        required=True,
+        ondelete="cascade",
+    )
+    payload = fields.Json()
+    result = fields.Text()
+
+    def _execute(self, **kwargs):
+        self.ensure_one()
+        payload = self.ai_bridge_id._prepare_payload(
+            self.env[self.model_id.model].browse(self.res_id), **kwargs
+        )
+        response = requests.post(
+            self.ai_bridge_id.url,
+            json=payload,
+            auth=self._get_auth(),
+            headers=self._get_headers(),
+            timeout=30,  # Default timeout, can be overridden by _execute_kwargs
+            **self._execute_kwargs(**kwargs),
+        )
+        response.raise_for_status()
+        self.result = response.content()
+        self.payload = payload
+
+    def _execute_kwargs(self, timeout=False, **kwargs):
+        result = {}
```

Comment:
self.ensure_one() ?

Reply:
It is not really necessary for private functions.... anyway, I can do that.